src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -468,7 +486,6 @@ def create_lnd_container(
             f"--bitcoind.zmqpubrawblock={bitcoind_rpc_host}:{tank.zmqblockport}",
             f"--bitcoind.zmqpubrawtx={bitcoind_rpc_host}:{tank.zmqtxport}",
             f"--rpclisten=0.0.0.0:{tank.lnnode.rpc_port}",
-            f"--externalhosts={lightning_dns}",
```

Comment:
might reconsider keeping this. from the doc:

```
      --externalhosts=                                       Add a hostname:port that should be periodically resolved to announce IPs for. If a port is not specified, the default (9735) will be used.
```

Reply:
Hmm, not sure what to do here, as we won't know the ip address until after startup in our classic, re-occurring chicken-and-egg problem.

Reply:
but `lightning_dns` is a domain name isn't it? Sounds to me like lnd will resolve the IP sometime after startup